pytorch_lightning/core/lightning.py

```diff
@@ -440,6 +440,20 @@ def __sync(
     def __check_none(name: str, value: Any, _) -> Any:
         raise ValueError(f'`self.log({name}, {value})` was called, but `None` values cannot be logged')
 
+    def log_grad_norm(self, grad_norm_dict: Dict[str, torch.Tensor]) -> None:
+        """Override this method to change the default behaviour of ``log_grad_norm``.
+
+        Args:
+            grad_norm_dict: Dictionary containing current grad norm metrics
+
+        Example::
+
+            # DEFAULT
+            def log_grad_norm(self, grad_norm_dict):
+                self.log_dict(grad_norm_dict, on_step=False, on_epoch=True, prog_bar=False, logger=True)
+        """
+        self.log_dict(grad_norm_dict, on_step=True, on_epoch=True, prog_bar=True, logger=True)
```

Comment:
while not BC, is `prog_bar=False` a better default?

Reply:
@carmocca Any thoughts on this one ? I see value in both.